athena-cloudera-impala/src/main/java/com/amazonaws/athena/connectors/cloudera/ImpalaFederationExpressionParser.java

```diff
@@ -17,7 +17,7 @@
  * limitations under the License.
  * #L%
  */
-package com.amazonaws.athena.connectors.hortonworks;
+package com.amazonaws.athena.connectors.cloudera;
```

Comment:
why did it change package?

Reply:
For some reason, it was in the hortonworks package even though it is for cloudera. I assume it was someone's mistake when copy pasting.

Reply:
There are a few files in the impala connector where it had to be imported because the package name was wrong. Ex: athena-cloudera-impala/src/main/java/com/amazonaws/athena/connectors/cloudera/ImpalaRecordHandler.java